virtual_oss.8

```diff
@@ -138,6 +138,9 @@ Set control device name.
 .It Fl P Ar devname
 Set playback DSP device only.
 Specifying /dev/null is magic and means no playback device.
+Specifying a
```

Comment:
You may want to add an example in the EXAMPLES section.

Reply:
Agreed -- I've added one.